docker-compose.test.yml

```diff
@@ -50,3 +53,8 @@ services:
         ! dig @server -p 53 fail02.dnssec.works +dnssec +multi | tee /dev/stderr | grep NOERROR
         ! dig @server -p 53 fail03.dnssec.works +dnssec +multi | tee /dev/stderr | grep NOERROR
         ! dig @server -p 53 fail04.dnssec.works +dnssec +multi | tee /dev/stderr | grep NOERROR
+
+        # 9.9.9.11 is the public DNS resolver from Quad9 that explicitly does pass ECS information
+        dig @server -p 53 TXT o-o.myaddr.l.google.com +short @9.9.9.11 | tee /dev/stderr | grep edns0-client-subnet
+        # 9.9.9.9 is the public DNS resolver from Quad9 that explicitly does NOT pass ECS information
+        ! dig @server -p 53 TXT o-o.myaddr.l.google.com +short @9.9.9.9 | tee /dev/stderr | grep edns0-client-subnet
```

Comment:
@TomTonic for some reason this command is returning the same `edns0-client-subnet` values as `9.9.9.11` when I would have expected it to be just the A record as per your [comment](https://github.com/klutchell/unbound-docker/issues/165#issuecomment-1692268276).

Any idea why that may be?

Reply:
Do I need to test `9.9.9.9` before testing `9.9.9.11`?

Reply:
I assume it is because of the two `@server` parameters in line 60. Maybe it queries the first server (‘@server’) instead of the second server (‘@9.9.9.9‘), and the first server has ECS enabled in both cases?  (Sorry I can‘t test. On vacation with a very lousy internet connection.)

Reply:
Oh duh, I didn't even notice that. I'll just test the success case for now on the local test server.